### ISSUE TITLE
rake apt:build stalls because of lintian's check with encrypted zip files

### ISSUE THREAD

**Issue by kenhys** [clear-code/td-agent-builder#37]
# Problem

rake apt:build stalls because of lintian's check with encrypted zip files

# Expected

`rake apt:build` succeeds without stalling.

# Actual

`rake apt:build` stalls. It is caused by stopped lintian check process.

```
 184023 pts/0    S+     0:09 lintian --profile td-agent/ubuntu td-agent_3.7.0-1_amd64.changes
 185063 pts/0    S+     0:00 unzip -t /tmp/lintian-pool-cdSp7Kwv7Q/pool/t/td-agent/td-agent_3.7.0-1_amd64_binary/unpacked/opt/td-agent/lib/ruby/gems/2.4.0/gems/rubyzip-1.3.0/test/data/zipWithEncryption.zip

  17:07

        |-containerd-+-containerd-shim-+-build.sh---debuild-+-lintian --profi---unzip
        |            |                 |                    `-tee
        |            |                 `-9*[{containerd-shim}]
        |            `-24*[{containerd}]
```

This situation occurs on Ubuntu 20.04(focal)

# Workaround

It seems that this is the bug of lintian.
It was already reported as https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=935292 and fixed in lintian/2.67.0.

Workaround is:
* Use lintian on testing(bulleseye) or sid
apt install -t testing lintian on buster
* on Ubuntu, fixed version is not released yet.

**Comment by ashie** [comment]
I think another workaround is that removing the /opt/td-agent/lib/ruby/gems/2.4.0/gems/rubyzip-1.3.0/test/data/zipWithEncryption.zip from the package until fixed lintian is released for Ubuntu Focal.
Probably it's not needed at runtime.

**Comment by kenhys** [comment]
It seems that another workaround is reasonable.

**Comment by kenhys** [comment]
https://github.com/clear-code/td-agent-builder/blob/master/td-agent/Rakefile#L269
I thought removing the file after install_gems is ok, but not work as expected. ???

**Comment by ashie** [comment]
This issue will be fixed by #44 via #52.
